<?xml version="1.0" encoding="UTF-8"?>
<!--
/*
 * rtty gui widget definition
 *
 * Copyright 2022 Marc CAPDEVILLE F4JMZ
 *
 * This is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; either version 3, or (at your option)
 * any later version.
 *
 * This software is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this software; see the file COPYING.  If not, write to
 * the Free Software Foundation, Inc., 51 Franklin Street,
 * Boston, MA 02110-1301, USA.
 */
-->
<ui version="4.0">
 <class>DockRTTY</class>
 <widget class="QDockWidget" name="DockRTTY">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>245</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>245</height>
   </size>
  </property>
  <property name="floating">
   <bool>false</bool>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>RTTY</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QCheckBox" name="rttyEnable">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Enable</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="reset">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="save">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Space freq</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="space_freq">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>75</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="minimum">
                 <double>-3000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>3000.000000000000000</double>
                </property>
                <property name="value">
                 <double>-85.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="swap">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>16</height>
               </size>
              </property>
              <property name="text">
               <string>&lt;-&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Mark freq</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="mark_freq">
                <property name="minimumSize">
                 <size>
                  <width>75</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="minimum">
                 <double>-3000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>3000.000000000000000</double>
                </property>
                <property name="value">
                 <double>85.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Baud rate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="baud_rate">
                <property name="minimumSize">
                 <size>
                  <width>75</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="minimum">
                 <double>1.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>2000.000000000000000</double>
                </property>
                <property name="value">
                 <double>45.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Parity</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="parity">
                <property name="minimumSize">
                 <size>
                  <width>75</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="currentIndex">
                 <number>-1</number>
                </property>
                <property name="frame">
                 <bool>true</bool>
                </property>
                <property name="modelColumn">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Mode</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="mode">
                <property name="minimumSize">
                 <size>
                  <width>75</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="currentIndex">
                 <number>-1</number>
                </property>
                <property name="frame">
                 <bool>true</bool>
                </property>
                <property name="modelColumn">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="text">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Courier 10 Pitch</family>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="documentTitle">
         <string>RTTY decode</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
